test/test_torch.py

```diff
@@ -2440,6 +2440,21 @@ def _generate_noncontiguous_input():
                     self.assertEqual(tensor.repeat(*repeat).numpy(),
                                      np.tile(tensor.numpy(), repeat))
 
+        @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+        def test_tile(self):
+            initial_shape = (8, 4, 3)
```

Comment:
For this test you'll want to enumerate a cross product of input shapes and reps values to test the fewer dims, more dims, and same dims cases. Also test for an empty input shape an an empty reps.

Reply:
Also, do the test with both a tensor constructed in that shape and the transpose of the same tensor.